app/models/status.rb

```diff
@@ -23,5 +23,6 @@ class Status < ApplicationRecord
                 removed: "removed",
                 discarded: "discarded",
                 superseded: "superseded",
-                scheduled: "scheduled" }
+                scheduled: "scheduled",
+                failed_to_publish: "failed_to_publish" }
```

Comment:
As discussed there is very little that is specific to scheduling as this status doesn't have a model. I'd hope it'd be compatible with async publishing too since the details is the scheduling model and that could be something different.